lib/msal-common/src/error/ClientAuthError.ts

```diff
@@ -169,10 +169,12 @@ export const ClientAuthErrorMessage = {
  * Error thrown when there is an error in the client code running on the browser.
  */
 export class ClientAuthError extends AuthError {
+    
+    static CLIENT_AUTH_ERR_NAME: string = "ClientAuthError";
```

Comment:
Can we put all these in an enum in Constants?

Reply:
I thought it would be easier to have them part of the error class instead (since we only reference them in that context). `Constants.ts` is more for generic strings used across the app.